### PR TITLE
Updated with the logo

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Updated it with the logo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Octokit - GitHub API Client Library for .NET [![Build Status](https://ci.appveyor.com/api/projects/status/github/octokit/octokit.net?branch=master)](https://ci.appveyor.com/project/Haacked15676/octokit-net)
 
+![logo](octokit.png)
```
Comment (Haacked) [inline README.md]: I love it! However, the logo is HUGE and takes up so much space. Let's use a normal image tag and size it to a reasonable size.

Comment (naveensrinivasan) [inline README.md]: How about this one http://haacked.com/images/haacked_com/WindowsLiveWriter/IntroducingOctokit.NET_D8EF/octokit-dotnet_2.png?

Comment (Haacked) [inline README.md]: :+1:

**Comment by naveensrinivasan** [comment]
Done!

**Pull request merged**